code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -50,17 +50,17 @@
 	path = /obj/item/clothing/head/beret/purple
 
 /datum/gear/head/beret/sec
-	display_name = "beret,security"
```

Comment:
Probably, but, not really an issue when a lot of loadout and preferences changes are happening with this update.